CorePreferences/src/au/gov/asd/tac/constellation/preferences/ApplicationPreferenceKeys.java

```diff
@@ -107,7 +107,7 @@ private static String getDefaultUserDir() {
     public static final String JUPYTER_NOTEBOOK_DIR_DEFAULT = new File(System.getProperty("user.dir")).getPath();
 
     /**
-     * Remember Save As location.
+     * Remember Open and Save location.
      */
     public static final String REMEMBER_SAVE_LOCATION = "rememberSaveLocation";
     public static final boolean REMEMBER_SAVE_LOCATION_DEFAULT = true;
```

Comment:
I feel like these should be renamed to `REMEMBER_OPEN_SAVE_LOCATION` and `REMEMBER_OPEN_SAVE_LOCATION_DEFAULT` to reflect that it is for both open and save now (not just save)

Reply:
@antares1470 I changed these (manually), but it threw an error, so I reverted them back. Is there a different way to rename?

Reply:
@Auriga2 I tried this locally on my machine and I didn't seem to have any such troubles. Did you refactor the variable name or were you manually changing it at all occurrences?

Reply:
I can't refactor, get this error:
![image](https://user-images.githubusercontent.com/64826928/128134846-3095f87f-0570-4f1c-b2cb-bdaafe7619e8.png)

So I manually edited. Could be a manual error that broke it. Can you see the prefs in the `Setup->Options->CONSTELLATION` tab after refactoring?

Reply:
Oh I can refactor, had to open the module before refactoring, as suggested by the message above. I'll see if it works now.

Reply:
Refactoring worked. I changed other occurrences as well.